extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -539,27 +539,11 @@ export async function promptConnectionStringPasswordAndUpdateConnectionString(co
 			}
 		}
 		// set connection string to not include the password if connection info does not include password, or user chooses to not include password, or authentication type is not sql login
-		let userPassword: string | undefined;
 		if (includePassword !== constants.yesString) {
 			connectionString = await vscodeMssqlApi.getConnectionString(connectionDetails, false, false);
-
-			// Ask user to enter password if auth type is sql login and password is not saved
-			if (connectionInfo.authenticationType === 'SqlLogin' && connectionInfo.password) {
-				userPassword = await vscode.window.showInputBox({
-					prompt: constants.enterPasswordPrompt,
-					placeHolder: constants.enterPasswordManually,
-					ignoreFocusOut: true,
-					password: true,
-					validateInput: input => input ? undefined : constants.valueMustNotBeEmpty
-				});
-				if (userPassword) {
-					// if user enters password replace password placeholder with user entered password
-					connectionString = connectionString.replace(constants.passwordPlaceholder, userPassword);
-				}
-			}
 		}
 
-		if (includePassword !== constants.yesString && !userPassword && connectionInfo?.authenticationType === 'SqlLogin') {
+		if (includePassword !== constants.yesString && connectionInfo?.authenticationType === 'SqlLogin') {
```

Comment:
You can just add this as another if inside the one on line 542 to consolidate the logic for when the user doesn't save the password.